librz/core/cbin.c

```diff
@@ -37,30 +37,57 @@ static void pair(const char *key, const char *val) {
 	if (!val || !*val) {
 		return;
 	}
-	rz_cons_printf("%-" PAIR_WIDTH "s%s\n", key, val);
 }
 
-static void pair_bool(PJ *pj, const char *key, bool val) {
+static char *get_coloured(const char *flag, bool use_color) {
+	if (use_color == true) {
+		if (strncmp(flag, "true", 5) == 0) {
+			return rz_str_newf(Color_GREEN "true" Color_RESET);
+		} else {
+			return rz_str_newf(Color_RED "false" Color_RESET);
+		}
+	} else {
+		if (strncmp(flag, "true", 5) == 0) {
+			return rz_str_newf("true");
+		} else {
+			return rz_str_newf("false");
+		}
+	}
+}
+
+static void pair_bool(bool use_color, RzTable *t, PJ *pj, const char *key, bool val, RzList *row_list) {
 	if (pj) {
 		pj_kb(pj, key, val);
 	} else {
-		pair(key, rz_str_bool(val));
+		RzTableColumnType *typeString = rz_table_type("bool");
+		const char *b = get_coloured(rz_str_bool(val), use_color);
+		pair(key, b);
+		rz_table_add_column(t, typeString, key, 0);
+		rz_list_append(row_list, rz_str_newf("%s", b));
```

Comment:
`typeString` or `b` or both may be leaking, and probably you should check if they are NULL pointers, since you are allocating them.